src/rgw/rgw_s3select_private.h

```diff
@@ -208,6 +208,7 @@ class RGWSelectObj_ObjStore_S3 : public RGWGetObj_ObjStore_S3
   std::string output_row_delimiter;
   std::string m_start_scan;
   std::string m_end_scan;
+  std::string m_output_json_format;
```

Comment:
it is better to use class-enum for CSV, JSON (in the future... probably a binary format)